tests/periph_timer_short_relative_set/README.md

```diff
@@ -0,0 +1,29 @@
+# Periph Timer Test for short timer_set() values
+
+## About
+
+This test exposes if timer_set() with very low values (down to zero) underflows.
+For each value from 100 to 0, it'll try to set a timer with that length.
+In human terms, that should trigger instantly.
```

Comment:
I propose to phrase "In human terms, that should trigger instantly" more precisely. It actually depends on the configuration of the timer (TEST_TIMER_FREQ), in case of the default xtimer config this *should* be 1 MHz.

Reply:
I've elaborated a little.